pytorch_lightning/trainer/trainer.py

```diff
@@ -518,6 +520,9 @@ def dispatch(self):
 
     def run_stage(self):
         results = None
+
+        self._run_stage_setup()
```

Comment:
should this call instead `run_setup_hook`?

where `run_setup_hook` includes `self.profiler_connector.setup()`

and we remove the `run_setup_hook` call in fit

Reply:
This should be done later to resolve ddp_spawn setup issues.